ucp/core.py

```diff
@@ -624,6 +656,24 @@ async def recv(self, buffer, tag=None):
             self.abort()
         return ret
 
+    @nvtx_annotate("UCXPY_AM_RECV", color="red", domain="ucxpy")
+    async def am_recv(self):
+        """Receive from connected peer.
+        """
+        if self.closed():
+            raise UCXCloseError("Endpoint closed")
+        log = "[Recv AM #%03d] ep: %s" % (self._recv_count, hex(self.uid),)
```

Comment:
```suggestion
        log = "[AM Recv #%03d] ep: %s" % (self._recv_count, hex(self.uid),)
```

Reply:
Fixed in https://github.com/rapidsai/ucx-py/pull/701/commits/a3f3b0870d650785cb7bb14993a2da6ee2aaf47e